### DISCUSSION TITLE
Thunderbird 98.0b2 - Google calendar not show..

### DISCUSSION THREAD

**Discussion by MD299** [kewisch/gdata-provider#407] [Support] [answered]
Hi, 
this add-on is perfectly, till new version of TB 98.0b2.
Now I cannot synchro events from Google calendar to see in TB.
But when I create new event from TB to Google Calendar, in Google I can see this event...
Ih TB console I can see messages like this...see attachments...
Best regards, M
![tb01](https://user-images.githubusercontent.com/98303920/154707003-27df3022-2ea1-4ac6-9d91-4da6697ec791.jpg)
D

**Comment by c765f614** [comment]
Got exactly the same issue as MD299, nothing visiable in TB 98.0b2 wheryby it all worked perfectly before. Somewhere a change in TB what is causing this maybe?
![Schermafbeelding 2022-02-27 203901](https://user-images.githubusercontent.com/31311312/155897155-568a0be7-8558-48ef-a61d-a53093b7e798.png)

**Comment by wlittle8** [comment]
Same here. New events show up in GC but not in TB. Also, notifications do not go away when dismissed. I'm running TB 98.0b2 and Provider for Google Calendar 91.0.4.

**Comment by harshness** [comment]
I am having this problem with beta 3 as well.

Calendar: TypeError: this._stream is undefined   
Symbol.asyncIterator resource:///modules/calendar/utils/calIteratorUtils.jsm:263    
start resource:///modules/CalReadableStreamFactory.jsm:274    
createCombinedReadableStream resource:///modules/CalReadableStreamFactory.jsm:271    
start chrome://calendar/content/calendar-invitations-manager.js:207    
calendar-invitations-manager.js:124

**Comment by kewisch** [comment, marked as answer]
Thank you for flagging this. I was aware, though didn't have time to follow the beta channel. I'll be releasing this as 102.0.0 soon with a fix.
